faunadb-java-dsl/src/test/java/com/faunadb/client/dsl/DslSpec.java

```diff
@@ -17,57 +18,81 @@
 import org.joda.time.Duration;
 import org.joda.time.Instant;
 import org.joda.time.LocalDate;
-import org.junit.BeforeClass;
+import org.junit.Before;
 import org.junit.Rule;
 import org.junit.Test;
 import org.junit.rules.ExpectedException;
 
 import java.util.Arrays;
+import java.util.List;
 import java.util.Random;
 
 import static com.faunadb.client.query.Language.Action.CREATE;
 import static com.faunadb.client.query.Language.Action.DELETE;
 import static com.faunadb.client.query.Language.*;
 import static com.faunadb.client.query.Language.TimeUnit.*;
 import static com.faunadb.client.types.Codec.*;
+import static com.faunadb.client.types.Value.NullV.NULL;
+import static com.google.common.base.Functions.constant;
+import static com.google.common.util.concurrent.Futures.catching;
+import static com.google.common.util.concurrent.Futures.transformAsync;
 import static org.hamcrest.CoreMatchers.*;
 import static org.hamcrest.collection.IsArrayContainingInOrder.arrayContaining;
 import static org.hamcrest.collection.IsCollectionWithSize.hasSize;
 import static org.hamcrest.collection.IsIterableContainingInOrder.contains;
 import static org.joda.time.DateTimeZone.UTC;
 import static org.junit.Assert.assertThat;
 
-public class ClientSpec extends FaunaDBTest {
+import static java.lang.String.format;
+
+public abstract class DslSpec {
+
+  protected static final String ROOT_TOKEN = EnvVariables.require("FAUNA_ROOT_KEY");
```

Comment:
I don't feel like it's DslSpec's responsibility to deal with environment variables.
The whole spec is very abstract and has nothing to do with those variables.